sydent/sydent.py

```diff
@@ -124,6 +124,21 @@
         'email.smtppassword': '',
         'email.hostname': '',
         'email.tlsmode': '0',
+        # When a user is invited to a room via their email address, that invite is
+        # displayed in the room list using an obfuscated version of the user's email
+        # address. These config options determine how much of the email address to
+        # obfuscate. Note that the '@' sign is always included.
+        #
+        # If the given username or domain is shorter than the threshold defined here,
+        # the string is then redacted based on its length. This ensure that a full email
```

Comment:
Updated - does what I wrote make sense?

Reply:
> If the string is longer than a configured limit below, it is truncated to that limit...

sounds a lot like

> If the given username or domain is shorter than the threshold defined here,
> the string is then redacted...

But otherwise lgtm

Reply:
Maybe it is unclear?

> If the string is longer than a configured limit below, it is truncated to that limit...

This means with limit 5, string "1234567890" would be truncated to "12345..."

> If the given username or domain is shorter than the threshold defined here,
the string is then redacted...

This means with limit 5, string "123" would become "1...".

The first is talking about truncating a long string, and the second is talking about redacting a short string.

Reply:
Sorry, I explained my point badly. What I meant was that:

> If the string is longer than a configured limit below, it is truncated to that limit
> with '...' added.

seems unnecessary because if we're redacting based on the length then the string is shorter than the threshold then the string can't be longer than the threshold.

It feels to me like this sentence is describing a step that's part of what happens when "the string is then redacted based on its length", which makes it confusing imho.

Reply:
Ahh, I get what you mean now. Removed the first line :slightly_smiling_face: